### PR TITLE
Remove login

### PR DESCRIPTION
**Pull request by FokkeZB**
No longer needed for Titanium CLI.

### CODE REVIEW

**Comment by skypanther** [comment]
@FokkeZB Any chance you can resolve the merge conflicts so that @tonylukasavage can merge this? It's a blocker for using grunt-titanium at this point.

**Comment by FokkeZB** [comment]
@skypanther of course! ready to merge @tonylukasavage!

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-titanium",
   "description": "grunt plugin for titanium CLI",
-  "version": "0.3.2",
+  "version": "0.3.3",
```
Comment (tonylukasavage) [inline package.json]: @FokkeZB bad open source etiquette, really shouldn't change version numbers in PRs. Remove that please.

File: package.json [hunk 2]
```diff
@@ -48,6 +48,6 @@
     "fs-extra": "^0.11.0",
     "lodash": "~2.4.1",
     "recursive-readdir": "^1.2.0",
-    "titanium": ">3.2.0"
```
Comment (tonylukasavage) [inline package.json]: @FokkeZB unless I'm missing something, it seems to me that there's no reason to remove support for earlier versions of the titanium cli here. If I'm understanding, ti cli 4+ doesn't require a login. Rather than removing the login check and removing support for ti cli 3.x.x, you should probably instead make checks in the code to see what version of the ti cli is installed as a dependency, and then decide whether or not to skip the login check.

In short, there's no sense in removing support for earlier versions of the ti cli if a simple conditional can avoid it, rather then stripping out functionality.

Comment (FokkeZB) [inline package.json]: But it's not up to the user which version is installed right? With `>3.2.0` wouldn't NPM just always install the latest available?

File: package.json (hunk 1 above)
Comment (FokkeZB) [inline package.json]: Yes sir!

File: package.json (hunk 2 above)
Comment (skypanther) [inline package.json]: The version needs to be bumped up. If left as `">3.2.0"` grunt-titanium seems to use the old CLI, which itself requires users to log in. Thus `grunt` still fails with an `[ERROR] Authentication required` message. By setting this to `">4.0.0"` it works just fine with Fokke's other changes. The need to log in was removed in the 4.0 CLI.

**Comment by FokkeZB** [comment]
Closing for inactivity
